src/NUnitConsole/nunit3-console.tests/ConsoleRunnerTests.cs

```diff
@@ -11,10 +11,15 @@ namespace NUnit.ConsoleRunner.Tests
 {
     class ConsoleRunnerTests
     {
-        [Test]
+        [Test, Ignore("Needs to be rewritten")]
```

Comment:
> but that does not seem to make us want to fix them

The reason is that 

> they will have to use some judgement as to how ignore is being used in each situation and whether an issue is called for.

I have no idea what the rationale is behind someone else marking a test ignored. On the other hand, an issue marked `bug` and `confirm` have no such shades of meaning. Either case, it's a matter of education and establishing practices of regularly sweeping through with an eye to things like this. There are other things in issues as well to sweep for, like abandoned or un/mistriaged issues, so having practices like this all in one place makes issues a good candidate for tracking IMO.